internal/build/docker.go

```diff
@@ -375,6 +375,9 @@ func readDockerOutput(reader io.Reader) (*json.RawMessage, error) {
 			return nil, fmt.Errorf("decoding docker output: %v", err)
 		}
 
+		// TODO(Han): make me smarter! 🤓
+		logger.Get(ctx).Info(fmt.Sprintf("%+v\n", message))
```

Comment:
wonder if we need Infof and friends

Reply:
whoops, maybe I just named it poorly
`func (l logger) Info(format string, a ...interface{})`